imagetest/test_suites/disk/disk_readwrite_test.go

```diff
@@ -0,0 +1,33 @@
+//go:build cit
+// +build cit
+
+package disk
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+func TestDiskReadWrite(t *testing.T) {
+	utils.WindowsOnly(t)
```

Comment:
Why is this windows only? Cant' we just change the paths for linux and test on both?

Reply:
Expanding the test is much more working than just moving it over in this PR, but I think there should be a TODO comment for this at least.

Reply:
I added the linux test for disk, mostly a copy of the part of the hotattach test which reads the file.